apps/protocol-frontend/src/components/ImportHistoryTable.tsx

```diff
@@ -73,7 +73,7 @@ const ImportHistoryTable = () => {
         }) => {
           return (
             row.original.import_status.name === 'Completed' &&
-            userDaos?.get(getValue())?.membershipStatus?.name === 'admin' && (
+            userDaos?.get(getValue())?.membershipStatus?.name === 'Admin' && (
```

Comment:
We should move these to an enum. This can be done in a separate ticket.

Reply:
Totally agree. Especially so that it prevents mistakes like we see here ('admin' -> 'Admin')